chore(operator): release notes for Armory Operator 1.6.0 (#1149)
armory/docs: content/en/armory-enterprise/release-notes/rn-armory-operator/armory-operator-v1-6-0.md

---
title: v1.6.0 Armory Operator
toc_hide: true
version: 01.06.00
description: Release notes for Armory Operator v1.6.0
---

## 2022/01/26 Release Notes

## Security

Armory scans the codebase as we develop and release software. For information about CVE scans for this release, contact your Armory account representative.

## Known issues

No known issues.

## Highlighted updates

> This release improves configuration options for Sidecars with support for `resources` as well as `readinessProbe` and `startupProbe`.

Reviewer: ```suggestion
Operator now supports the following configuration options for sidecars: `resources`,`readinessProbe`,  and `startupProbe`.
```
